backend/config/curation-api.yml

```diff
@@ -288,13 +288,41 @@ paths:
           $ref: "#/components/responses/401"
         "403":
           $ref: "#/components/responses/403"
-
   /v1/collections/{collection_id}/datasets:
+    post:
+      summary: Create a Dataset ID to be used in a future upload.
+      description: |
+        Create an new empty Dataset and return the `dataset_id`. The `dataset_id` is used when uploading a Dataset file 
+        to S3 to identify what Dataset the file belongs to.
```

Comment:
Replace this with the verbiage that was in this section when it was further below and was introduced by the curator tag removal PR 👇

Reply:
actually not just the `description`. Everything needs to be copied over, including the `dataset_id` -> `id` change.